### PR TITLE
[samsungtv] Add UE50MU6179 and additional documentation

### PR DESCRIPTION
**Pull request by aliewen**
Added UE50MU6179 as I was able to use a number of channels. Let me know if any additional details or proof are needed.

Added a table with valid values for some common apps for users' convenience as well as a reference to the log file to find any additionally installed apps.

Side note: Key codes for remote control simulation might be useful as well since online documentation seems rather weak. I'm happy to contribute as I learn more - just unsure how this should be structured, by TV type or remote type.

Signed-off-by: Andreas Liewen github.andreas@liewen.net

### CODE REVIEW

File: bundles/org.openhab.binding.samsungtv/README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ Tested TV models:
 | LE40D579    | PARTIAL | Supported channels: `volume`, `mute`, `channel`, `keyCode`, `sourceName`,  `programTitle`, `channelName`,  `power`                                     |
 | LE40C650    | PARTIAL | Supported channels: `volume`, `mute`, `channel`, `keyCode`, `brightness`, `contrast`, `colorTemperature`, `power` (only power off, unable to power on) |
 | UE55LS003   | PARTIAL | Supported channels: `volume`, `mute`, `sourceApp`, `url`, `keyCode`, `power`, `artMode`                                                                |
+| UE50MU6179  | PARTIAL | Supported channels: `volume`, `mute`, `power`, `keyCode`, `channel`, `sourceApp`, `url` (at least)                                                                |
```
Comment (J-N-K) [inline bundles/org.openhab.binding.samsungtv/README.md:24]:
```suggestion
| UE50MU6179  | PARTIAL | Supported channels: `volume`, `mute`, `power`, `keyCode`, `channel`, `sourceApp`, `url` |
```

File: bundles/org.openhab.binding.samsungtv/README.md [hunk 2]
```diff
@@ -86,3 +87,18 @@ String  TV_KeyCode       "Key Code"                            (gLivingRoomTV)
 Switch  TV_Power         "Power"                               (gLivingRoomTV)   { channel="samsungtv:tv:livingroom:power" }
 Switch  TV_ArtMode       "Art Mode"                            (gLivingRoomTV)   { channel="samsungtv:tv:livingroom:artMode" }
 ```
+
+### Apps
+
+List of known apps and the respectve name that can be passed on to the `sourceApp` channel. Values are confirme to work on UE50MU6179.
```
Comment (J-N-K) [inline bundles/org.openhab.binding.samsungtv/README.md:93]:
```suggestion
List of known apps and the respective name that can be passed on to the `sourceApp` channel.
Values are confirme to work on UE50MU6179.
```

Use a new line for each sentence

Comment (aliewen) [reply bundles/org.openhab.binding.samsungtv/README.md:93]: Looks like I can't do any changes to what was submitted or proposed, So if you don't mind, kindly fix the typo in "Values are confirme to work on UE50MU6179." for me as well. Might save us from starting over with a new PR. Sorry I'm new to Github.

Comment (J-N-K) [reply bundles/org.openhab.binding.samsungtv/README.md:93]: done

**Comment by J-N-K** [comment]
Sign-off in first post

**Pull request merged**